src/consensus/params.h

```diff
@@ -176,8 +176,8 @@ struct Params {
     int nHighSubsidyFactor{1};
 
     std::map<LLMQType, LLMQParams> llmqs;
-    LLMQType llmqChainLocks;
-    LLMQType llmqForInstantSend{LLMQ_NONE};
+    LLMQType llmqTypeChainLocks{LLMQ_NONE};
```

Comment:
scripted-diff fails because of this addition.
To run scripted diff `~/workspace/dash$ ./contrib/devtools/commit-script-check.sh upstream/develop..HEAD`

Reply:
Ahhh 🙈